coalib/output/dbus/DbusDocument.py

```diff
@@ -124,7 +124,7 @@ def Analyze(self):
                     continue
 
                 if any([fnmatch(self.path, file_pattern)
-                        for file_pattern in path_list(section["files"])]):
+                        for file_pattern in glob_list(section["files"])]):
```

Comment:
I think this should also go with the processing commit since it's almost the same (file collection, but instead of `path_list`, we use `glob_list`).